api/modelgeneration/modelgeneration.go

```diff
@@ -116,25 +118,29 @@ func (c *Client) GenerationInfo(modelUUID string) (model.GenerationSummaries, er
 	if result.Error != nil {
 		return nil, errors.Trace(result.Error)
 	}
-	return generationInfoFromDTO(result.Applications), nil
+	return generationInfoFromResult(result.Generation), nil
 }
 
 func argForModel(modelUUID string) params.Entity {
 	return params.Entity{Tag: names.NewModelTag(modelUUID).String()}
 }
 
-func generationInfoFromDTO(apps []params.GenerationApplication) model.GenerationSummaries {
-	appDeltas := make([]model.GenerationApplication, len(apps))
-	for i, a := range apps {
+func generationInfoFromResult(res params.Generation) model.GenerationSummaries {
+	appDeltas := make([]model.GenerationApplication, len(res.Applications))
+	for i, a := range res.Applications {
 		appDeltas[i] = model.GenerationApplication{
 			ApplicationName: a.ApplicationName,
 			Units:           a.Units,
 			ConfigChanges:   a.ConfigChanges,
 		}
 	}
+	gen := model.Generation{
+		Created:      time.Unix(res.Created, 0).Local(),
```

Comment:
Any particular reason for using `.Local()` instead of `.UTC()`.  Timezones are evil after all 🙈

Reply:
Changed it to be user driven as per status.